dygraphs/dygraphs.d.ts

```diff
@@ -0,0 +1,1272 @@
+// Type definitions for dygraphs
+// Project: http://dygraphs.com
+// Definitions by: Dan Vanderkam <http://danvk.org>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace dygraphs {
+  type DataArray = number[][];
+
+  // TODO: check for another DT definition to use here.
```

Comment:
I'm not really familiar with `typings`, but it is common practice to use triple-slash references in DefinitelyTyped, so please do so. Use `/// <reference path="../google.visualization/google.visualization.d.ts" />` and I'll convert this to `/// <reference types="google.visualization" />` when porting to the `types-2.0` branch.